### PR TITLE
Remove unused proxy api

### LINKED ISSUE #72: Remove the unnecessary proxy API
Since the swarm-node provide the full-featured TangleID API, the TangleID/CoreAPI.js should call these API directly, not just using backend server to forward these API request to swarm-node.

Excepted output: 
 -  Remove the unnecessary proxy API in `server.js` [[link]](https://github.com/TangleID/TangleID/blob/cb071ab6be6674afb6b95da0e7a164db1dda86fa/server.js#L29)

### PR DESCRIPTION
**Pull request by HexRabbit**
With the implementation of `CoreAPI.js`,
we don't have to proxy TangleID api requests anymore.

Close #72

### CODE REVIEW

**Comment by jserv** [comment]
Move `Close #72` to the end of commit messages.

**Comment by HexRabbit** [comment]
@ender503 The API call by `Login` action is not in the scope of this pull request, so I'll keep it.

**Comment by jserv** [comment]
I am confused by inconsistent upper/lower case about "login" and "Login". @ender503, can you clarify?

**Comment by ender503** [comment]
The function name starts with the upper case which means constructor, otherwise regular function.
In this case, the function name should be started with lowercase. 👍

**Comment by jserv** [comment]
@ender503, send another pull request to address upper/lower case use scenario in file `STYLE-GUIDE.md`.

File: actions/fetchClaims.js [hunk 1]
```diff
@@ -1,27 +0,0 @@
-import {
-  CALL_API,
-  FETCH_CLAIMS_REQUEST,
-  FETCH_CLAIMS_FAILURE,
-  FETCH_CLAIMS_SUCCESS,
```
Comment (ender503) [inline actions/fetchClaims.js]: These constants no longer use, is it okay to clean up them? 🤔

Comment (ender503) [reply actions/fetchClaims.js]: I create a new PR to clean these unsued constants.

**Pull request merged**